tests/tests.js

```diff
@@ -172,7 +172,7 @@ define(['jquery', 'title', 'archive', 'zimArchive', 'zimDirEntry', 'util', 'geom
                 equal(title._name, "Diego_Velázquez", "Name of the title is correct");
                 start();
             };
-            localEvopediaArchive.getTitleByName("Diego_Velázquez").then(callbackFunction);
+            localEvopediaArchive.getTitleByName("A/Diego_Velázquez").then(callbackFunction);
```

Comment:
You seem to prefix Evopedia titles with ZIM namespaces : are you sure? I suppose it should be applied only on ZIM titles?